src/chainstate/stacks/miner.rs

```diff
@@ -5794,19 +5794,31 @@ pub mod test {
     }
 
     pub fn make_coinbase(miner: &mut TestMiner, burnchain_height: usize) -> StacksTransaction {
```

Comment:
I think it's time for #3159 -- this is another PR that would not have any changes in `chainstate::stacks::miner` except for the fact that the test framework is there.

Reply:
Either is fine, but the follow-on PR must be merged soon after this PR.

Reply:
I've opened the PR that does this here: https://github.com/stacks-network/stacks-blockchain/pull/3180.  It's against this PR, so let's get it reviewed and merged first and then we can merge this PR to `next`.